cmd/statusd/main.go

```diff
@@ -27,6 +30,8 @@ var (
 	httpEnabled    = flag.Bool("http", false, "HTTP RPC endpoint enabled (default: false)")
 	httpPort       = flag.Int("httpport", params.HTTPPort, "HTTP RPC server's listening port")
 	ipcEnabled     = flag.Bool("ipc", false, "IPC RPC endpoint enabled")
+	cliAddr        = flag.String("cli", "", "Enable debugging CLI connection for <address>:<port>")
```

Comment:
I can think of valid use cases for where client address could be unknown. What if we just provide a bool flag enable/disable and allow the server to accept any connection?

Reply:
Reason behind it has been security. Removing this and also defining a fixed port would surely make the code more simple.